transforms/transforms.go

```diff
@@ -97,6 +97,22 @@ var JoinType = joinStruct{Left: joinType{0}, Inner: joinType{1}}
 
 // Join returns a TransformFunc that joins Rows with another table using the specified join type.
 func Join(rightTable optimus.Table, leftHeader string, rightHeader string, join joinType) optimus.TransformFunc {
+	return JoinCustom(rightTable, leftHeader, rightHeader, join, mergeRows)
```

Comment:
This shouldn't be in this PR. This was a mistake from my original commit. Can you revert this file to the same as master?